utils/mtpoe-ctrl/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018 adron@yapic.net
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtpoe-ctrl
+PKG_SOURCE_VERSION:=9c42b74274cca10810363dbbb6d161bfb9cedb7e
+PKG_MIRROR_HASH:=1ac2902161ba4bb1e8af397aa48821ccea95740199b1c155f98161c7ff3a93df
+PKG_VERSION:=1.15
+PKG_RELEASE:=18
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prudy/mtpoe_ctrl.git
```

Comment:
I wonder, why did you forked the original repository and there isnt anything new.

Reply:
There are changes to adapt for the v4 of the protocol. They are awaiting with th PR to original author repo. Till then it points to my fork.